linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -458,14 +462,29 @@ pipeline {
                     "${LISAV2_PARAMS}" +
                     " -CustomKernel 'localfile:./scripts/package_building/${env.BUILD_NUMBER}-${env.BRANCH_NAME}-${env.KERNEL_ARTIFACTS_PATH}/*/${env.PACKAGE_TYPE}/*.${env.PACKAGE_TYPE}'" +
                     " -OsVHD '${HYPERV_VHD_PATH}'" +
-                    " -XMLSecretFile '${HyperV_Secrets_File}'"
+                    " -XMLSecretFile '${HyperV_Secrets_File}'" +
+                    " -ExitWithZero"
                 )
             }
           }
           post {
             always {
               junit "Report\\*-junit.xml"
               archiveArtifacts "TestResults\\**\\*"
+              script {
+                    def return_values = powershell returnStdout: true, script: '''$content = [xml](Get-Content ".\\Report\\*-junit.xml")
```

Comment:
groovy and java use camelCase variable names for local variables. replace all the new variable names accordingly.